### PR TITLE
US 045 get dashboard information

### PR DESCRIPTION
**Pull request by rocioferreiro**
## Description

> Created a "DashboardInformation" object, that has every information necessary to create the dashboard, total amount of active loans can be achieved by summing up delayed, withdrawal and readyForWithdrawal amounts. For line chart, a map was used to get the label and the amount together.
 
## ClubHouse Link

> https://app.clubhouse.io/bibliotecame/story/302/dashboard-admin
## Checklist
- [X] I've created and run successfully the related unit tests
- [X] I've run successfully every test in the repo
- [X] I've test the related endpoints with Postman
- [X] This pull request has 'sprint_x' as the base branch
- [X] This pull request has a descriptive title and description
- [X] I have merged the current sprint into my branch before pushing 
- [X] I've added the QA Leader as a reviewer

### CODE REVIEW

**Review by FacundoBocalandro — CHANGES_REQUESTED** [review]
Me tira este error. Parece que javafx dejo de estar incluido en el sdk (https://stackoverflow.com/questions/27178895/cannot-resolve-symbol-javafx-application-in-intellij-idea-ide). ¿Hay forma de cambiarlo?

[
error java
](url)

**Comment by rocioferreiro** [comment]
@FacundoBocalandro ahi lo cambie

File: src/test/java/bibliotecame/back/LoanTests.java [hunk 1]
```diff
@@ -676,12 +676,11 @@ public void testDelayedCheckAndNotify() throws NotFoundException {
 
         loanController.setWithdrawDate(loan.getId());
         loan = loanService.getLoanById(loan.getId());
-        assertFalse((Boolean) loanController.checkDelayedLoans().getBody());
```
Comment (FacundoBocalandro) [inline src/test/java/bibliotecame/back/LoanTests.java:679]: Porqué hiciste cambios acá?

Comment (rocioferreiro) [reply src/test/java/bibliotecame/back/LoanTests.java:679]: Lost test de loan no estaban teniendo en cuenta el caso de que pueden aparecen otros loans con estas características en otros tests. Lo cambie para que no se rompa con los que cree para dashboard

**Review by FacundoBocalandro — APPROVED** [review]
LGTM

**Review by LucasManza — APPROVED** [review]

**Pull request merged**